Assets/Scripts/Utility/AssetInjection/SoundReplacement.cs

```diff
@@ -78,7 +78,7 @@ public static bool TryImportMidiSong(string filename, out byte[] songBytes)
         /// <summary>
         /// Import sound data from modding locations as an audio clip.
         /// </summary>
-        private static bool TryImportAudioClip(string name, string extension, out AudioClip audioClip)
+        private static bool TryImportAudioClip(string name, string extension, out AudioClip audioClip, bool streaming = false)
```

Comment:
I'm ok with this. Just a small note: the convention is to place the out parameter in the last position. I would prefer if you move `streaming` before it and just pass true and false respectively.

Reply:
Done. Somehow I was expecting to have more places to update...